packages/x-components/src/x-modules/scroll/components/scroll-to-top.vue

```diff
@@ -107,7 +117,7 @@
      * @internal
      */
     protected get isThresholdReached(): boolean {
-      return this.useThresholdStrategy && this.scrollTop > (this.thresholdPx as number);
+      return this.useThresholdStrategy && this.scrollData.position > this.thresholdPx!;
```

Comment:
This `useThresholdStrategy` computed property name is confusing isn't it? Maybe something like `isThresholdANumber`

Reply:
I don't think so. This component has two ways of working, rendering itself when a certain threshold is reached, or when the user is about the end the scroll and keeps scrolling down. I don't even know if this last way is necessary, as it does not work properly with the infinite scroll, maybe it only makes sense for pagination. 

The `useThresholdStrategy` name exactly tells this, if the component should render after a threshold. `isThresholdANumber` says nothing about the intention for me.